### DISCUSSION TITLE
Unable to Enable V3D - Additional Solution

### DISCUSSION THREAD

**Discussion by USAFPride** [rtl-airband/RTLSDR-Airband#333] [Ideas]
I recommend adding the following as an additional trouble shooting step. My rPi3 2 and 3B would throw the error until I did the following:

sudo raspi-config
advanced options -> GL driver -> legacy

**Comment by om3wuc** [comment]
I just dicovered same thing when installing on Zero W with correct rpi1 platform switch....
